client/src/models/observation.js

```diff
@@ -0,0 +1,55 @@
+import { Model } from 'radiks';
+
+// Observation models all observational questions we ask users
+export default class Observation extends Model {
+    static className = 'Observation';
+    static schema = {
+        coronavirus: {
```

Comment:
@SomeMoosery Why did you add all these information into one model? I thought there is a daily model and a non-changing model

Reply:
Modeled the data more closely to models used in the medical field after a conversation with @ngiangre - to the best of my knowledge "Observation" is a more scientifically correct umbrella which contains breakdowns (as I have here) within it - and physical versus non-physical observations is a more useful to way to break out data.

What do you think? Do you see a downside in modeling the data in this way?

Reply:
Users won't update their height or weight on a daily basis, therefore, finding the latest entered height or weight requires to go through all observations (backwards until the value was found)

Reply:
Good point. I can create a new PR moving age, height, and any other one-time attributes into the demographics collection, as these are really just demographics